templates/default/logback.xml.erb

```diff
@@ -43,8 +50,39 @@
     </encoder>
   </appender>
 
+  <% if node.cassandra.logback.debug.enable -%>
```

Comment:
Apparently the "dot access" feature will be gone in Chef 13: https://travis-ci.org/michaelklishin/cassandra-chef-cookbook/builds/188491258. Let's switch to "bracket access". Also, I'm not 100% sure if the CI failure is due to these changes but it seems to be that way.